test/jdk/java/nio/channels/vthread/BlockingChannelOps.java

```diff
@@ -22,17 +22,25 @@
  */
 
 /**
- * @test
+ * @test id=default
  * @bug 8284161
  * @summary Basic tests of virtual threads doing blocking I/O with NIO channels
  * @enablePreview
  * @library /test/lib
  * @run testng/othervm/timeout=300 BlockingChannelOps
+ */
+
+/**
+ * @test id=useDirectRegister
+ * @bug 8284161
+ * @summary Basic tests of virtual threads doing blocking I/O with NIO channels
```

Comment:
Hello Aleksey, I see that previously the `@bug` and `@summary` was only there on one test definition. So maybe remove this duplicated `@bug` and `@summary` from this new test definition?
Other than that, this test change looks OK to me.

Reply:
On a related note - I had a brief look at the jtreg docs including the FAQ https://openjdk.org/jtreg/faq.html and I don't see any mention about how it treats multiple test definitions in a single source file, when it comes to running them concurrently. But I believe you have more experience with it and have already tested the difference in parallelism, so I think what you suggest here is fine.

Reply:
Yes, this is a usual thing to replicate the test sections. It is also a usual practice to copy-paste the entirety of run definition/config.